src/olympia/amo/management/__init__.py

```diff
@@ -0,0 +1,144 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from celery import chord, group
+
+from olympia.amo.utils import chunked
+
+
+class ProcessObjectsCommand(BaseCommand):
+    """
+    A generic command base class to run a task on objects.
+
+    Inherit from it and implement `get_model()` and `get_tasks()`.
+    """
+
+    def get_model(self):
```

Comment:
nit, but I'd just had this as a property in the subclass declaration as it doesn't need to be dynamically determined.

Reply:
I originally had this as a property but I wanted to be able to raise `NotImplementedError` to get an explicit exception if someone tried to use that class directly...